rpa-home/modules/ROOT/pages/anypoint-migration-overview.adoc

```diff
@@ -0,0 +1,41 @@
+= Anypoint Platform Migration Overview
+
+Starting July 1st, RPA Manager integrates with Anypoint Platform to unify the login experience and provide support for external identities (IDP).
+
+If your organization acquired the MuleSoft Automation Bundle after the integration, your account is not affected by the migration because your organization is already in Anypoint Platform. 
+
+The migration process for existing RPA customers starts on July 1st and is carried out automatically in different waves. To prepare for the migration, ensure you follow the <<pre-migration, Pre-migration Steps>>.
+
+For additional questions, contact us at rpa_migration_support@salesforce.com.
+
+== Summary of Changes
+
+After your organization is migrated to Anypoint Platform, the following changes apply:  
+
+* Users are required to reconfigure 2FA for their Anypoint Account.
+* Users log in to RPA Manager, RPA Builder, and RPA Recorder using their Anypoint Platform account. 
+* Administrators use Access Management to manage RPA users, teams, and permissions.
+* RPA Manager Roles and User groups are replaced by teams in Anypoint Platform. 
+* API keys are deprecated and replaced by Connected Apps. 
+
+[[pre-migration]]
+== Pre-migration Steps
+
+To prepare your organization for the migration, ensure that all your RPA Manager users have an account in Anypoint Platform or the external identity provider (IDP) your organization uses to log in.
+
+* Federated orgs
++
+If your Anypoint Organization is using an external identity provider (IDP), the administrator must ensure the RPA users have a matching account and the proper access.
+
+* Non-Federated orgs
++
+The migration is automatic and MuleSoft matches RPA accounts and Anypoint accounts based on the email address. 
++
+** If the RPA user already has an Anypoint account with the same email, the account linking is automatic. After the migration, the user accesses RPA resources using the Anypoint account, retaining the same level of permissions. 
+** If the RPA user has multiple Anypoint accounts with the same email address, the user is prompted to select the account to link when logging in.
```

Comment:
the duplicity is for RPA users and not Anypoint accounts:
If there is more than one RPA user with the same email address as the Anypoint account used to login, the user is prompted to select the account to link when logging in. (pro tip: If one of these RPA users had the same username as the one in Anypoint, the selection will be done automatically)